src/bootstrap-table.js

```diff
@@ -2397,7 +2424,7 @@ class BootstrapTable {
     this.initPagination()
     this.initBody()
     if (this.options.showColumns) {
-      const $items = this.$toolbar.find('.keep-open input:not(".toggle-all")').prop('disabled', false)
+      const $items = this.$toolbar.find('.keep-open input[type="checkboxc"]:not(".toggle-all")').prop('disabled', false)
```

Comment:
There is a typo in checkbox word

Reply:
At this moment the live example is not working

Reply:
dammit... fixed :D
The live example is still not working because of the caching issues of jsdelivr.